allure-generator/src/test/javascript/stubs/handlebars-compiler.js

```diff
@@ -5,8 +5,6 @@ const path = require('path');
 const fs = new MemoryFS();
 const file = process.argv[2];
 
-console.log('SETUP HOHOHO');
-
```

Comment:
yep, that is my fault. I can't get them working again after merging generator & web together. Could you please take a look?